frontend/src/views/pages/PaymentPage.tsx

```diff
@@ -65,27 +85,29 @@ export const PaymentPage = () => {
       ...member,
       paid: member.id.toString() === event.target.value,
     }));
-
     setMemberExpense(updatedPaid);
   };
 
-  const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
-    event.preventDefault();
+  const createExpense = handleSubmit(async (data: Expense) => {
+    // Here you have access to data which includes amount, description and others that you registered
     const resultInsertExpense = await insertExpense();
     if (resultInsertExpense) navigate("/history");
-  };
+  });
 
   console.log(error);
 
   const insertExpense = async () => {
+    if (description === "") {
+      setError("Description cannot be empty");
+      return false;
+    }
```

Comment:
please also check if the description is not a blank like " " and "mulitple spaces".

Reply:
Okay. I add .trim() method.